pregred-gui/src/app/pages/home/Home.js

```diff
@@ -1,9 +1,255 @@
-import React from 'react';
+import React,{useState} from 'react';
 
 import './Home.css';
+import { Modal, Button } from 'antd';
 
 function Home() {
-  return <h1> PregRed App is Running - Home </h1>;
+  const [visible, setVisible1] = useState(false);
+  const [visible2, setVisible2] = useState(false);
+  const [visible3, setVisible3] = useState(false);
+  const [visible4, setVisible4] = useState(false);
+  const [visible5, setVisible5] = useState(false);
+  const [visible6, setVisible6] = useState(false);
+  const [visible7, setVisible7] = useState(false);
+  return(
+    <div className="container-home">
+
+         <div className="container-home-div1">
+           {/* <h3 className="homepage-para-header">Abstract</h3> */}
+            <p className="homepage-para">
+            The pregnancy period is one of the most important periods in the life of a woman.
+            The health of the baby is the most crucial factor in that period.
+            <br/>
+            PregRed will help you to detect
+            how the pre-pregnancy factors affect the baby’s health,
+            get knowledge about pre-pregnancy health impacts and can meet health
+            professionals.
+            </p>
+
```

Comment:
As we planned, all information should be navigated to another page and breadcrumb should be there.. should not be shown in modal . isn't it?